### ISSUE TITLE
[1.12.1] [2.6] Sometimes plugin throws an Exception

### ISSUE THREAD

**Issue by benazarenko** [filbert66/EnchantLimiter#5]
```
[23:41:43 WARN]: [EnchLimiter] Task #13 for EnchLimiter v2.6 generated an exception
java.lang.NullPointerException: null
	at com.yahoo.phil_work.enchlimiter.EnchLimiter$1enchantGiveback.run(EnchLimiter.java:290) ~[?:?]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:58) ~[patched_1.12.1.jar:git-Paper-"6d9375d2"]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:356) [patched_1.12.1.jar:git-Paper-"6d9375d2"]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:833) [patched_1.12.1.jar:git-Paper-"6d9375d2"]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:424) [patched_1.12.1.jar:git-Paper-"6d9375d2"]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:767) [patched_1.12.1.jar:git-Paper-"6d9375d2"]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:665) [patched_1.12.1.jar:git-Paper-"6d9375d2"]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_131]
```
I think it's a bug. Hope it'll be fixed soon.

**Comment by filbert66** [comment]
Yep. When you enchant without lapis. Is that permissible in creative mode?
I can fix very soon.

**Comment by filbert66** [comment]
Fixed in 2.6.1
